hazelcast-client-new/src/main/java/com/hazelcast/client/impl/DefaultClientExtension.java

```diff
@@ -69,6 +69,9 @@ public SerializationService createSerializationService() {
             SerializationServiceBuilder builder = new DefaultSerializationServiceBuilder();
             SerializationConfig serializationConfig = config.getSerializationConfig() != null ? config
                     .getSerializationConfig() : new SerializationConfig();
+            if (version > 0) {
```

Comment:
I see below that -1 is accepted; this can be fine.. but can you add it to the javadoc?